pymeasure/adapters/prologix.py

```diff
@@ -107,15 +110,66 @@ def __init__(self, resource_name, address=None, rw_delay=0, serial_timeout=None,
                          **kwargs)
         self.address = address
         if not isinstance(resource_name, PrologixAdapter):
-            self.set_defaults()
+            self.auto = auto
+            self.eoi = eoi
+            self.eos = eos
+
+    @property
+    def auto(self):
+        """Configure Prologix GPIB controller to automatically address instruments
+        to talk after sending them a command in order to read their response. The
+        feature called, Read-After-Write, saves the user from having to issue read commands
+        repeatedly. This property enabled or disabled the Read-After-Write feature.
```

Comment:
```suggestion
        repeatedly. This property enables or disables this feature.
```
I think also valid inputs values and their meaning should be specified in the doc string.